### DISCUSSION TITLE
定时任务页面希望增加排序功能

### DISCUSSION THREAD

**Discussion by fmz200** [whyour/qinglong#534] [Ideas]
定时任务页面的任务名和任务两列要是增加排序按钮就好了，这样方便查看哪些任务重复了，哪些脚本重复了，用户体验更佳，哈哈哈，望采纳
![image](https://user-images.githubusercontent.com/57740735/128036230-ec9ea704-df29-4af9-b0be-8a9fd333a92f.png)

**Comment by ZeroMah** [comment]
希望删除定时任务的时候，增加（勾选后删除脚本文件）按钮.望采纳，谢谢
